documentation/head/replication.md

```diff
@@ -4,39 +4,37 @@ todo
 
 # Overview
 
-Postgres 9.4 (released in December 2014) introduced a new feature logical replication. Logical replication allow
-stream changes from database in real-time to external system.
+Postgres 9.4 (released in December 2014) introduced a new feature called logical replication. Logical replication allows
+changes from database to be streamed in real-time to external system. The difference between streaming replication and
+logical replication is that logical replication sends data over in a logical format whereas streaming replication sends data over in a binary format. Additionally logical replication can send over a single table, or database. Streaming replication was all or nothing.
```

Comment:
>The difference between streaming replication and
 +logical replication is that logical replication sends data over in a logical format whereas streaming replication sends data over in a binary format.

I thinks streaming replication term applies to both physical replication and logical replication.